Userland/Libraries/LibGUI/EmojiInputDialog.cpp

```diff
@@ -23,69 +23,25 @@
 #include <LibGUI/ScrollableContainerWidget.h>
 #include <LibGUI/TextBox.h>
 #include <LibGUI/Toolbar.h>
-#include <LibGfx/Bitmap.h>
-#include <LibGfx/Font/Emoji.h>
 
 namespace GUI {
 
 struct EmojiCateogry {
```

Comment:
Just noticed there's a typo in this struct name.

Reply:
Added a commit to fix it since it was my typo originally anyways